greptimedb/data_size.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+curl -s --fail http://localhost:4000/v1/sql \
+    -H 'Content-Type: application/x-www-form-urlencoded' \
+    -d "sql=SELECT sum(r.sst_size) as data_size FROM information_schema.REGION_STATISTICS r LEFT JOIN information_schema.TABLES t on r.table_id = t.table_id WHERE t.table_name = 'jsontable'"  \
```

Comment:
We are fetching `sst_size` which (I think) stands for "sorted string table". Are we sure that at this point no data is in memory tables/buffers?